hack/after-build/verify-symbols.sh

```diff
@@ -24,11 +24,17 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 # add other BADSYMBOLS here.
-BADSYMBOLS="testing[.]"
+BADSYMBOLS=(
+  "httptest"
+  "testify"
+  "testing[.]"
+)
+# Join symbols with OR '\|' pattern
+PATTERN="$(printf '\|%s' "${BADSYMBOLS[@]}" | tail -c +3)"
 
 # b/c hyperkube binds everything simply check that for bad symbols
-if [[ $(nm ${KUBE_OUTPUT_HOSTBIN}/hyperkube | grep "${BADSYMBOLS}") ]]; then
-  echo "output binaries contain bad symbols \"${BADSYMBOLS}\""
+if nm ${KUBE_OUTPUT_HOSTBIN}/hyperkube | grep "${PATTERN}"; then
+  echo "output binaries contain bad symbols"
```

Comment:
By removing the brackets in the `if` statement it outputs all the matches now. I removed printing the symbols since I thought the matches were more useful, but if you still think we should have both I'll add it back.